openapi-v2.yaml

```diff
@@ -1533,6 +1533,8 @@ definitions:
         properties:
           azure:
             $ref: "#/definitions/AzureToken"
+          google:
```

Comment:
No strong preference on my end, but maybe this should be "gcp" since we use the "aws" acronym for the other credentials?

Reply:
That seems like a reasonable idea. I'll leave this open until tomorrow for any other feedback and submit it then.